src/libs/actions/Timing.js

```diff
@@ -51,11 +51,11 @@ function end(eventName, secondaryName = '') {
             return;
         }
 
-        API.write('SendPerformanceTiming', {
+        API.read('SendPerformanceTiming', {
```

Comment:
By making it a read it does not get retried nor saved in onyx nor anyhing like that, which for a stats command makes total sense.

Reply:
Yeah, this is tracking metrics on speed, it's not really important to save every piece of data. And by removing this it means it does not block other actually important requests (both in normal conditions and especially after coming back online since that's when the app becomes unresponsive or outdated)
So it's kind of the solution. I do need to investigate why this failure was not triggering our reauthentication mechanism, when it should've, but that's work for another PR.